Helper/Cart.php

```diff
@@ -1354,6 +1354,7 @@ function ($item) use ($imageHelper, &$totalAmount, &$diff, $storeId, $currencyCo
                 $product['unit_price']   = CurrencyUtils::toMinor($unitPrice, $currencyCode);
                 $product['quantity']     = round($item->getQty());
                 $product['sku']          = trim($item->getSku());
+                $product['quote_item_id']= $item->getId();
```

Comment:
I am not sure it's the best way. We need `quote_item_id` only when we remove the item from the cart, but this line includes it to bolt cart as well.
As I see right now we use only reference, quantity, and quote_item_id. I suggest creating another helper that returns only the information we needed.
It's not only the way, we can use some other approaches as well, but lets save bolt cart unaltered.

Reply:
It is too specific, so I create a method in UpdateCartCommon class, please review again